.github/workflows/build_examples.yml

```diff
@@ -0,0 +1,62 @@
+name: Wildfly Operator example images build
+
+on:
+  repository_dispatch:
+    types: [build_operator_examples]
+
+env:
+  LANG: en_US.UTF-8
+  S2I_URI: https://api.github.com/repos/openshift/source-to-image/releases/latest
+  QUAY_USERNAME: ${{ secrets.QUAY_USERNAME }}
+  QUAY_PASSWORD: ${{ secrets.QUAY_PASSWORD }}
+  EVENT_SOURCE: ${{ github.event.client_payload.source }}
+  EVENT_TYPE: ${{ github.event.action }}
+
+jobs:
+  build:
+    if: github.event.action == 'build_operator_examples'
+    name: Build Wildfly Operator examples
+    runs-on: ubuntu-latest
+    steps:
+    - name: Print info
+      run: echo "${EVENT_TYPE} received from ${EVENT_SOURCE}"
+    - name: Pull latest wildfly-s2i image
+      run: docker pull quay.io/wildfly/wildfly-centos7:latest
+    - name: Login quay.io
+      run: docker login -u="${QUAY_USERNAME}" -p="${QUAY_PASSWORD}" quay.io
+    - name: Install s2i binary
```

Comment:
This might be simplified using https://github.com/redhat-actions/s2i-build. Have you tried it?

Reply:
I looked at it and it's not usable: it does not allow to specify a Git ref (it's expected to build the checked out repo which is not our case here)

Reply:
Thanks for let me know, I was trying it just now :)